ls1intum/Artemis
# remove buildUrlWithParams from globa-utils

### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server 

### Motivation and Context


The `buildUrlWithParams` utility function is broken when using more than one parameter. Furthermore it was only used with a single parameter. 

Also add a couple of small tests for `Utils.ts`.

### Description

I removed `buildUrlWithParams` and adapted the existing code. I also added a couple of tests in `utils.spec.ts`

### Steps for Testing

Simply save an assessment for a file upload and programming exercise and make sure it works correctly

### Test Coverage



global-utils.ts : 100%
utils.ts : 97.14%
